fe-vite-react/src/NewsItem.jsx

```diff
@@ -0,0 +1,19 @@
+
+const NewsItem =({article}) => {
+    return (
+        <div className = "news-item">
```

Comment:
Lots of divs appearing in this project. I think we could use more semantic html tags for a11y purposes